sklearn/manifold/_mds.py

```diff
@@ -386,7 +386,7 @@ def __init__(self, n_components=2, *, metric=True, n_init=4,
 
     @property
     def _pairwise(self):
-        return self.kernel == "precomputed"
+        return self.dissimilarity == "precomputed"
```

Comment:
I suppose that this attribute was not tested because `kernel` is not an (hyper)parameter in `sklearn.manifold.MDS`.

Reply:
I do not know why this was not tested. There was nothing in the test suite that actually used the `_pairwise` property.